docs/use_cases.md

```diff
@@ -0,0 +1,94 @@
+# Use cases
```

Comment:
This is repetead here, but it's already in main. It's here because of some git merge that I did in a wrong way, probably.

If it's better, I can close that PR and reopen another one.

Reply:
As long as it merges it is fine.